src/types.ts

```diff
@@ -517,7 +517,7 @@ export type Device = Readonly<{
 }>;
 
 export type UIChildrenComponents = {
-  [k in "FooterCenter"]?:
+  [k in "FooterCenter" | "Menu" | "Menu.Item"]?:
```

Comment:
```suggestion
  [k in "FooterCenter" | "Menu"]?:
```

`Menu.Item` is not a top-level children compo.